.gitignore

```diff
@@ -271,3 +271,4 @@ paket-files/
 # Environment Specific app settings
 appsettings.*.json
 secrets.config
+launchSettings.json
```

Comment:
Hey Arabella,

This change shouldn't be done here in this PR. Also, I'm pretty sure that `.gitignore` only applies to files that are not yet being tracked by git. `launchSettings.json` is already tracked by git, so this change wouldn't solve the issue anyway.

Reply:
I see. I just restored the `.gitignore` and `launchSettings.json` to their original states. So, next time when I want to push something to git, should I use `git add src` so that the changes I made in `launchSettings.json` wouldn't been added.